utilities/titandb/blob_format.h

```diff
@@ -159,7 +177,7 @@ class BlobFileMeta {
 // checksum             : fixed32
 struct BlobFileFooter {
   // The first 64bits from $(echo titandb/blob | sha1sum).
-  static const uint64_t kMagicNumber{0xcd3f52ea0fe14511ull};
+  static const uint64_t kFooterMagicNumber{0x2be0a6148e39edc6ull};
```

Comment:
Interestingly I rerun the command `echo titandb/blob | sha1sum` and it give me a different hash. Despite it affects nothing, I update it anyway.